docs/src/components/navigation/componentPreviewTabs.tsx

```diff
@@ -73,7 +73,7 @@ const tabStyles = {
 const tabPanelContentStyles = {
     display: 'flex',
     flexDirection: 'column',
-    height: '100%',
+    height: (theme: Theme) => `calc(100vh - ${theme.spacing(8)})`,
```

Comment:
Height shall stay 100%. 

The thing to change should be minHeight, or else you get strange things like this:
 
<img width="1339" alt="image" src="https://user-images.githubusercontent.com/8997218/183915304-e1eba8f8-3e56-4db7-a6a8-ca8d8a079628.png">

Reply:
> Height shall stay 100%.
> 
> The thing to change should be minHeight, or else you get strange things like this:
> 
> <img alt="image" width="1339" src="https://user-images.githubusercontent.com/8997218/183915304-e1eba8f8-3e56-4db7-a6a8-ca8d8a079628.png">

Thanks for this suggestion :). Done the changes also updated other readme with material UI URL.